src/lib/serviceRoutinesV2/getEntities.cpp

```diff
@@ -70,6 +70,11 @@ std::string getEntities
   string    idPattern  = ciP->uriParam["idPattern"];
   string    q          = ciP->uriParam["q"];
 
+  if (ciP->uriParam["options"] == "count")
```

Comment:
`if (ciP->uriParam["options"] == "count")` seems to be something to be fixed (to use uriParamOptions[]), as I understand that it would fails if the client uses for examples `...?options=count,otherop`.